libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/location/LocationComponent.kt

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.navigation.dropin.component.location
+
+import android.annotation.SuppressLint
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.whenResumed
+import com.mapbox.maps.MapView
+import com.mapbox.maps.plugin.LocationPuck2D
+import com.mapbox.maps.plugin.locationcomponent.location
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.dropin.R
+import com.mapbox.navigation.dropin.lifecycle.UIComponent
+import kotlinx.coroutines.launch
+
+@SuppressLint("MissingPermission")
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+class LocationComponent(
+    private val mapView: MapView,
+    private val lifecycleOwner: LifecycleOwner,
+    private val locationViewModel: LocationViewModel
+) : UIComponent() {
+
+    override fun onAttached(mapboxNavigation: MapboxNavigation) {
+        super.onAttached(mapboxNavigation)
+
+        coroutineScope.launch {
+            lifecycleOwner.whenResumed {
```

Comment:
This is the fix!

Reply:
Instead of waiting for the onStart, waiting for the first location because that seems to be the real event that causes the location to not activate